packages/firestore/test/integration/api/validation.test.ts

```diff
@@ -1003,125 +1003,99 @@ apiDescribe('Validation:', (persistence: boolean) => {
+          .where('foo', inOp, [1, 2])
+          .where('foo', inOp, [2, 3])
+      ).to.throw("Invalid query. You cannot use more than one 'in' filter.");
 
-        expect(() =>
-          db
-            .collection('test')
-            .where('foo', arrayContainsAnyOp, [2, 3])
-            .where('foo', 'array-contains', 1)
-        ).to.throw(
-          "Invalid query. You cannot use 'array-contains' filters with " +
-            "'array-contains-any' filters."
-        );
-      }
-    );
+      expect(() =>
+        db
+          .collection('test')
+          .where('foo', arrayContainsAnyOp, [1, 2])
+          .where('foo', arrayContainsAnyOp, [2, 3])
+      ).to.throw(
+        "Invalid query. You cannot use more than one 'array-contains-any'" +
+          ' filter.'
+      );
 
-    validationIt(
-      persistence,
-      'cannot have array-contains-any filter with in filter.',
-      db => {
-        expect(() =>
-          db
-            .collection('test')
-            .where('foo', arrayContainsAnyOp, [2, 3])
-            .where('foo', inOp, [2, 3])
-        ).to.throw(
-          "Invalid query. You cannot use 'in' filters with " +
-            "'array-contains-any' filters."
-        );
+      expect(() =>
+        db
+          .collection('test')
+          .where('foo', arrayContainsAnyOp, [2, 3])
+          .where('foo', inOp, [2, 3])
+      ).to.throw(
+        "Invalid query. You cannot use 'in' filters with " +
+          "'array-contains-any' filters."
+      );
 
-        expect(() =>
-          db
-            .collection('test')
-            .where('foo', inOp, [2, 3])
-            .where('foo', arrayContainsAnyOp, [2, 3])
-        ).to.throw(
-          "Invalid query. You cannot use 'array-contains-any' filters with " +
-            "'in' filters."
-        );
-      }
-    );
+      expect(() =>
+        db
+          .collection('test')
+          .where('foo', inOp, [2, 3])
+          .where('foo', arrayContainsAnyOp, [2, 3])
+      ).to.throw(
+        "Invalid query. You cannot use 'array-contains-any' filters with " +
+          "'in' filters."
+      );
 
-    // This is redundant with the above tests, but makes sure our validation
-    // doesn't get confused.
-    validationIt(
-      persistence,
-      'cannot have array-contains, array-contains-any, and in filter.',
-      db => {
-        expect(() =>
-          db
-            .collection('test')
-            .where('foo', inOp, [2, 3])
-            .where('foo', 'array-contains', 1)
-            .where('foo', arrayContainsAnyOp, [2])
-        ).to.throw(
-          "Invalid query. You cannot use 'array-contains-any' filters with " +
-            "'in' filters."
-        );
+      // This is redundant with the above tests, but makes sure our validation
+      // doesn't get confused.
+      expect(() =>
+        db
+          .collection('test')
+          .where('foo', inOp, [2, 3])
+          .where('foo', 'array-contains', 1)
```

Comment:
Hrm...  shouldn't this fail to compile since you're not using `arrayContainsAnyOp`?

Reply:
`array-contains` has been released already, so this test is checking that `in` with `array-contains-any` still fails even if its separated by an `array-contains`.

Reply:
Oh, right! Brain hiccup. :)